### PR TITLE
Doc updates 2024 06 10

### PR DESCRIPTION
**Pull request by brainJamStark**

### CODE REVIEW

**Review by digiserg — APPROVED** [review]

File: docs/overview/motivation.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ To view the journey we have embarked on since 2017 have a look [here](https://ax
 
 ## Motivation
 
-AxonOps has been developed and actively maintained by [digitalis.io](https://digitalis.io), a company providing managed services for Apache Cassandra™ and other modern distributed data technologies.
+AxonOps has been developed and actively maintained by [axonops limited](https://axonops.com), a company providing managed services for Apache Cassandra™ and other modern distributed data technologies.
```
Comment (rgooding) [inline docs/overview/motivation.md:26]: `axonops limited` should probably be capitalised here as `AxonOps Limited`

**Review by rgooding — CHANGES_REQUESTED** [review]
Please see my comment below about captialisation of "AxonOps Limited"

File: docs/overview/motivation.md (hunk 1 above)
Comment (brainJamStark) [reply docs/overview/motivation.md]: Fixed

**Review by rgooding — APPROVED** [review]

**Review by hshimizu — APPROVED** [review]
approved

**Pull request merged**